book/src/external-provisioner.md

```diff
@@ -25,6 +25,8 @@ The deletion of a `PersistentVolumeClaim` object bound to a `PersistentVolume` c
 
 The CSI `external-provisioner` also supports the `Snapshot` DataSource. If a `Snapshot` CRD is specified as a data source on a PVC object, the sidecar container fetches the information about the snapshot by fetching the `SnapshotContent` object and populates the data source field in the resulting `CreateVolume` call to indicate to the storage system that the new volume should be populated using the specified snapshot.
 
+Cloning is also implemented by specifying a `kind:` of type `PersistentVolumeClaim` in the DataSource field of a Provision request.  It's the responsbility of the external-provisioner to verify that the claim specified in the DataSource object exists, is in the same storage class as the volume being provisioned and that the claim is currently `Bound`.
```

Comment:
Would it be useful to have a subsection just for DataSources?

Reply:
Yes, I'll look at doing that and adding sub-sections for the specific DS types.

Reply:
In this page can you also add a `### DataSources` subsection to put these 2 paragraphs under?  And then point to the data sources page?

Reply:
Yes, I have that actually; but it's not in this update.